lms/static/lms/js/require-config.js

```diff
@@ -64,15 +64,20 @@
             "tags-annotator": 'js/vendor/ova/tags-annotator',
             "diacritic-annotator": 'js/vendor/ova/diacritic-annotator',
             "flagging-annotator": 'js/vendor/ova/flagging-annotator',
-            "jquery-Watch": 'js/vendor/ova/jquery-Watch',
+            "jquery-Watch": "js/vendor/ova/jquery-Watch",
```

Comment:
I've converted the stray single quotes to be double quotes in this section, just to make them consistent.